testData/preferences/mary.json

```diff
@@ -4,10 +4,10 @@
             "gpii-default": {
                 "name": "Default preferences",
                 "preferences": {
-                    "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
+                    "http://registry.gpii.net/common/onscreenKeyboard": true,
```

Comment:
@kaspermarkus,

yeah, I'll only update the references to the `onscreenKeyboard` common term